spec/lib/api/redfish-1.0/account-service-spec.js

```diff
@@ -0,0 +1,236 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Redfish Account Service', function () {
+    var tv4;
+    var validator;
+    var redfish;
+    var waterline;
+    var Promise;
+    var template;
+    var fs;
+    var accountService;
+
+    // Skip reading the entry from Mongo and return the entry directly
+    function redirectGet(entry) {
+        return fs.readFileAsync(__dirname + '/../../../../data/templates/' + entry, 'utf-8')
```

Comment:
Using this constant about template path will be better: https://github.com/RackHD/on-core/blob/master/lib%2Fcommon%2Fconstants.js#L160